cilium/cmd/cleanup.go

```diff
@@ -42,7 +43,20 @@ var cleanupCmd = &cobra.Command{
 	},
 }
 
-var force bool
+var (
+	cleanAll bool
+	cleanBPF bool
+	force    bool
+)
+
+const (
+	allFlagName   = "all"
```

Comment:
let's make it more clear with "`cilium cleanup --all-state`" and  "`cilium cleanup --bpf-state`" instead of only `--all` and `--bpf`

Reply:
Fixed!